src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnection.cs

```diff
@@ -1395,6 +1412,12 @@ private bool TryOpenWithRetry(TaskCompletionSource<DbConnectionInternal> retry,
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml' path='docs/members[@name="SqlConnection"]/OpenWithOverrides/*' />
         public void Open(SqlConnectionOverrides overrides)
         {
+            if (_isExperimental)
+            {
+                _sqlConnectionX.Open();
```

Comment:
We would need to support `SqlConnectionOverrides` for EF Core - fail fast scenarios, so I would recommend building that in Open API from the beginning.